apps/omg/lib/omg/application.ex

```diff
@@ -20,9 +20,11 @@ defmodule OMG.Application do
 
   use Application
   alias OMG.Alert.AlarmHandler
+  require Logger
 
   def start(_type, _args) do
     :ok = AlarmHandler.install()
+    {_, _} = Logger.add_backend(Sentry.LoggerBackend)
```

Comment:
Not sure what the best way to do this is but their documentation suggests that this only needs to be loaded once otherwise it will return an error.

Reply:
Once per node?

Reply:
@InoMurko once per application instance yes

Reply:
Hmmm. That doesn’t make too much sense. Are you sure? Where did you notice that?
Logger is elixir built in logging library that supports multiple backends. But they normally don’t need to be installed moe then once.

Reply:
@InoMurko I don't think I explained myself well here. This line needs to only be included once, but I am unsure where is a good place within our (umbrella?) project cc https://github.com/getsentry/sentry-elixir/issues/345 for where I was looking for guidance.

Reply:
Aha! Alright, makes sense now. If you want to understand how umbrella applications are being divided between release modes (_wathcer_, _child_chain_) you may want to look at rel/config.exs. That would explain which umbrella applications are included in one or the other modus operandi. I suggest you either include in only in _omg_, or in both `omg_watcher_rpc` and `omg_child_chain_rpc`.